raiden-dapp/src/views/TransferRoute.vue

```diff
@@ -40,15 +43,47 @@ import { Token, TokenModel } from '@/model/types';
     NoChannelsDialog,
   },
   computed: {
+    ...mapState(['latestStateBackupReminder']),
     ...mapGetters(['tokens', 'channelWithBiggestCapacity', 'openChannels']),
   },
 })
 export default class TransferRoute extends Vue {
+  latestStateBackupReminder!: number;
   tokens!: TokenModel[];
+  currentTime: number = new Date().getTime();
```

Comment:
Question: When do you want to change this value? Why not getting the time when it is needed instead of when the class gets created? This could eventually be even wrong. In this case it probably would work. But it is an error prone pattern that you should not get used to. :wink: Just imagine it would be a `v-if` component similar. Just do it in a single line in `mounted()`.